integrated_channels/canvas/client.py

```diff
@@ -85,8 +91,20 @@ def create_content_metadata(self, serialized_data):
         )
 
         if not located_course:
+            LOGGER.info(
+                generate_formatted_log(
+                    'canvas',
+                    self.enterprise_configuration.enterprise_customer.uuid,
+                    None,
+                    edx_course_id,
+                    f'Creating new course with payload {desired_payload}',
+                )
+            )
             # Course does not exist: Create the course
-            status_code, response_text = self._post(self.course_create_url, serialized_data)
+            status_code, response_text = self._post(
```

Comment:
if we move the `restrict_enrollments_to_course_dates` addition to the canvas specific exporter than we can just continue to pass the serialized data through

Reply:
yeeeah, should be added? It'll technically be a part of the transformed data which I think gets added to the content metadata transmission audit record under `channel_medata`?

Reply:
@alex-sheehan-edx how's the new changes?